### ISSUE TITLE
Dark launch 1095B: Production launch

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#10627]
**BLOCKED by greenlight from 1095B team & Content/IA team**

### Staging/dark launch info
* **CMS link to content**: https://prod.cms.va.gov/records/download-your-irs-1095-b/ (formerly `/download-your-irs-1095-b-tax-form`)
* **Target production launch date:** During Feb 2025, for tax season
* **What is driving and/or blocking prod launch date?** Content review is only prod blocker, as far as we know
* **Technical point of contact** for vets-website React code: @mcommons10 
* **Strategic point of contact** for timing & approval to publish the node to Staging: @RLHecht , @mnorthuis 

## History
- #20516 
 

## Acceptance Criteria / Tasks
- [ ] Technical and/or strategic POCs have approved publish to prod
- [ ] Node ID is removed from [Staged Content entity subqueue](https://prod.cms.va.gov/admin/structure/entityqueue/staged_content/staged_content?destination=/admin/structure/entityqueue) - node id == 44346. (and queue is saved)
- [ ] Content release has run
- [ ] Page is live on VA.gov

**Comment by jilladams** [comment]
This 1095B project was sidelined due to the build team's contract transition in Dec 2022. As of Sept 2023, it's a side job for the health team: 
https://dsva.slack.com/archives/C02TUH5U3HS/p1694627391011989
> Hey Bill! This app is currently not under active development and is supported by the what "free cycles" the Health Team at OCTO has.
> This is a super low priority application and never really made it to a production read state (contracting timelines and other priorities took their toll )

@davidconlon FYI: this is the only live / current dark launched product. We made a page for the prior 1095B team, that team's contract ended, and it never prod launched. We could: 
1. Continue to leave that all as is. 
2. Un-dark launch and archive the dark launched node until further 1095B notice. 

If we wanted to go route 2, I think we'd need to review with Mark Dewey and Lauren Alexanderson. Not urgent, just bringing it up re: transitioning dark launches to CMS, in case we wanted to clean house first.

**Comment by FranECross** [comment]
Closing old iceboxed ticket.

**Comment by jilladams** [comment]
This node had been dark launched / staged this whole time. This week we got CAIA approval to unstage it: https://dsva.slack.com/archives/C01K37HRUAH/p1725578224168959, and that's been done. 

I also pinged the Veteran facing forms team to just check in about whether it will eventually be revived again or not: https://dsva.slack.com/archives/C044AGZFG2W/p1725982988927949

**Comment by jilladams** [comment]
What is dead may never die. 

Slack thread: https://dsva.slack.com/archives/C01K37HRUAH/p1739384336273409?thread_ts=1725578224.168959&cid=C01K37HRUAH

New dark launch request for the 1095B content: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/20516

**Comment by mcommons10** [comment]
Hi @mnorthuis @RLHecht @jilladams. We're not quiiiite ready for launch but very close. We've never launched a product on a Drupal page before so wanted to get some details squared away:
- We have a feature flag for our dynamic content. Does this mean that if we're doing a slow rollout, some users just won't be able to access our content until we're fully rolled out? There's no way to have the entire page wrapped up in that feature flag so it only shows up for the users that have the feature flag turned on?
- Once live in production, will this page automatically be added to the Records Hub and navigation? We weren't able to test that navigation path in staging I'm assuming because of the dark launch.

**Comment by jilladams** [comment]
As far as I understand it, you're right that a Drupal page is either on or off in Production. We can't do a gradual / % / users-based rollout of Drupal content pages.

Menus / nav changes: Content/IA will need to confirm based on what they know about where this page should appear in nav/menus. (If the menu item has been set on the node, then it should appear in related menus once it's published to prod. But if there are additional menus where it should appear vs. the Menu parent item on the node itself, those will need to be added / enabled separately.)

**Comment by RLHecht** [comment]
@mcommons10 we'll set up the left nav and menu info once we publish outside of the dark launch based on the IA spec. It will show in the records hub. We didn't set up any navigation now because we didn't want anything showing.

**Comment by mcommons10** [comment]
Thank you both! Sounds great.

**Comment by mcommons10** [comment]
@jilladams @RLHecht - We anticipate being ready to launch this page the week of March 24th. Should I just let you know when we have the green light? From when we say "go" to the page actually getting it published, about how long would that process take?

**Comment by RLHecht** [comment]
sounds good @mcommons10 When we publish in Drupal it can take between 60-90 minutes for it to show on production depending on where we are in the content build at that time.

**Comment by jilladams** [comment]
From ToT: launch scheduled for Wed 3/26.

**Comment by jilladams** [comment]
A Drupal Admin will need to remove this node from the Staging only queue when you all are ready to have it build a FE page. After that, a content release will need to run. This can be done alongside publishing the node.  @RLHecht @mcommons10  Please keep us posted on timing for launch.

**Comment by jilladams** [comment]
(Meaning: time of day on 3/26)

**Comment by RLHecht** [comment]
@mcommons10 are we still on track for wanting this live on March 26th at 2pm ET? If so, @jilladams does that mean I can publish around 1 pm at the same time a drupal admin would remove the node from the staging only queue?

**Comment by jilladams** [comment]
Yep, that works - I'll plan to remove this from the Staging queue at that time, after you give the greenlight on your publish, @RLHecht

**Comment by RLHecht** [comment]
@jilladams I just published this

**Comment by jilladams** [comment]
@RLHecht if we do _not_ see the page correctly going live after that content release, that will become a CMS team investigation, so you can ping the #sitewide-cms-platform channel and @ Grace and me for awareness.

**Comment by jilladams** [comment]
Page is live! https://www.va.gov/records/download-your-irs-1095-b/

Closing. Randi, feel free to reopen if you run into questions / issues.
